src/main/clojure/clara/rules/engine.cljc

```diff
@@ -215,7 +215,7 @@
      (->> node
          :children
          (map get-conditions-and-rule-names)
-         (reduce (partial merge-with conj) {})))))
```

Comment:
So this was wrong previously?

Reply:
This is a result of my merge snafu. This change as actually made for #379, in commit https://github.com/Provisdom/clara-rules/commit/3c9fc4e5a6bf1d9e2fd3a3c65d6fa78c1ddf0a29#diff-00da05e66726186c05209662d990b97c.

Reply:
Thanks for the reminder. I see now.